### DISCUSSION TITLE
Use many relays of the same type, Irf520

### DISCUSSION THREAD

**Discussion by anhpham140195** [theyosh/TerrariumPI#863] [Q&A] [answered]
Hello, currently I am having a problem when I connect the PI(zero 2 W) to 2 relays irf520 (gpio 13 and 37) 
In the TP4 relays were added successfully but it only works on one device attached to the relay. 
The other one doesn't and when I turn it on and off I hear a beeping sound
Thanks

**Comment by theyosh** [comment, marked as answer]
Hmm, this is a dimmer. And my documentation says all GPIO ports can be used. But I get the feeling that it will work better by using the following physical pin numbers: 12 and 33. Those are PWM pins, which I think will better work with dimmer relays. https://pinout.xyz/pinout/pwm

But that means that you have already have the max amount of dimmers you can use safely. There are two more PWM pins left, but they are combined, so despite the 4 pins, I can only use 2 for dimming....
